### PR TITLE
Add Travis builds

### PR DESCRIPTION
**Pull request by shadowhand**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - PHPUNIT_VERSION="^5.4"
+
+matrix:
+  include:
+    - php: 5.6
+      env:
+        - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+
+services:
+  - redis-server
```
Comment (Nyholm) [inline .travis.yml]: Why do we need the redis sever? Copy-paste issue?

Comment (shadowhand) [inline .travis.yml]: Yes.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - php: 5.6
+      env:
+        - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+
+before_script:
+  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
+  - travis_retry composer require phpunit/phpunit:${PHPUNIT_VERSION}
+
+script:
+  - vendor/bin/phpunit
```
Comment (Nyholm) [inline .travis.yml]: Lets use the global phpunit since we do not requiring it in composer.json

```
script:
  - vendor/bin/phpunit
```

Comment (shadowhand) [inline .travis.yml]: Is PHPUnit automatically installed in Travis builds? That's why I am installing it in `before_script`.

Comment (Nyholm) [inline .travis.yml]: I was wrong. It works anyways

**Comment by shadowhand** [comment]
Interesting, it appears that HHVM builds fail...

**Comment by Nyholm** [comment]
That is interesting. I'll merge this and create an issue for it.

**Pull request merged**